src/kafka_native/mod.rs

```diff
@@ -0,0 +1,222 @@
+use std::{borrow::Cow, collections::HashMap};
+use testcontainers::{
+    core::{ContainerPort, ContainerState, ExecCommand, WaitFor},
+    Image,
+};
+
+const KAFKA_NATIVE_IMAGE_NAME: &str = "apache/kafka-native";
+const KAFKA_IMAGE_NAME: &str = "apache/kafka";
+const TAG: &str = "latest";
+
+pub const KAFKA_PORT: u16 = 9092;
+
+const START_SCRIPT: &str = "/opt/kafka/testcontainers_start.sh";
+const DEFAULT_INTERNAL_TOPIC_RF: usize = 1;
+const DEFAULT_CLUSTER_ID: &str = "5L6g3nShT-eMCtK--X86sw";
+const DEFAULT_BROKER_ID: usize = 1;
+
+#[derive(Debug, Clone)]
+pub struct Kafka {
+    env_vars: HashMap<String, String>,
+    image_name: String,
+}
```

Comment:
Let's add some documentation, like this one: https://github.com/testcontainers/testcontainers-rs-modules-community/blob/71875a6483e43e4ff953e42c26e08825d2f7f0f3/src/rabbitmq/mod.rs#L6-L27

Including references to images